stub/src/main/java/io/grpc/stub/ClientCallStreamObserver.java

```diff
@@ -20,7 +20,8 @@
 
 /**
  * A refinement of {@link CallStreamObserver} that allows for lower-level interaction with
- * client calls.
+ * client calls. An instance of this class is obtained via {@link ClientResponseObserver}, or by
```

Comment:
`{@link ClientResponseObserver#beforeStart()"` to be more precisely?

Reply:
Does that verbosity actually help? There's only one method...